org.ekstep.sunbirdcommonheader-1.5/editor/service/service.js

```diff
@@ -0,0 +1,14 @@
+org.ekstep.services.sunbirdCommonConfiguration = new(org.ekstep.services.iService.extend({
```

Comment:
remove this service and `getFormConfigurations` api in meta-service.js inside content-editor.